### ISSUE TITLE
Request: Remove blank lines between single-line members.

### ISSUE THREAD

**Issue by MarkBrainfire** [codecadwallader/codemaid#777]
### Environment
- Visual Studio version: 2019 Community
- CodeMaid version: 11.2
- Code language: C#

### Description
I would like an option to remove blank lines BETWEEN single-line members (fields, properties, methods ... all single-line members).  C#/VS configuration makes possible single-line expression bodies in members.  Removing blank lines BETWEEN these members would result in more succinct code.  This is especially true when CodeLens is enabled--automatically adding reference links between members.  

I would like to retain (or add) 1 blank line between members where at least one of the members is a multiline member.  

### Steps to recreate
1. Arbitarily insert or remove lines and multiple lines between members.
1. Run the CodeMaid Cleanup 
2. Blank lines between single-line members are not removed and are inserted after single-line members -- even when followed by other single-line members.

Thank you!

Possible formatting BEFORE Cleanup.
![Possible BEFORE](https://user-images.githubusercontent.com/78389339/106534223-9b2fd100-64c1-11eb-80d3-0129596ef9c3.jpg)

Desired formatting AFTER Cleanup.
![Desired AFTER](https://user-images.githubusercontent.com/78389339/106534277-bef31700-64c1-11eb-9127-6b87e7e26463.jpg)

Actual formatting AFTER Cleanup.
![Actual AFTER](https://user-images.githubusercontent.com/78389339/106534317-d4684100-64c1-11eb-99c3-b1576073e876.jpg)

**Comment by codecadwallader** [comment]
Thanks for making this suggestion.  I think it may be a duplicate of #247 or #308 although with some additional configuration (dependent on size of member).

**Comment by jBachalo** [comment]
Yes. This is absolutely needed. The number one reason I use code formatters is blank line removal.
I need all blank lines within methods to be removed.
Surprised this isn't configurable. Otherwise fantastic tool!
